modules/ROOT/pages/discovering-and-consuming-apis.adoc

```diff
@@ -14,5 +14,5 @@ For more information, see the following topics:
 * xref:requesting-access-to-an-api-instance.adoc[Requesting Access to an API Instance]
 * xref:my-applications.adoc[My Applications]
 * xref:downloading-an-api-specification.adoc[Downloading a API Specification]
```

Comment:
I think it should say `Downloading an API Specification` instead

Reply:
yes, you are right